packages/vx-gradient/src/gradients/RadialGradient.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+type RadialGradientElementProps = React.SVGProps<SVGRadialGradientElement>;
+type StopElementProps = React.SVGProps<SVGStopElement>;
+
+export type RadialGradientProps = {
```

Comment:
Could also pick from `LinearGradientProps` to avoid writing the field documentation twice.

```ts
export type RadialGradientProps = Pick<
  LinearGradientProps,
  | 'id'
  | 'from'
  | 'to'
  | 'fromOffset'
  | 'fromOpacity'
  | 'toOffset'
  | 'toOpacity'
  | 'rotate'
  | 'transform'
  | 'children'
> &
  RadialGradientElementProps;
```